src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -132,20 +132,26 @@ on_exit() {
   rm -rf .nextflow
   rm nxf_runtime.config
 
-  # try uploading the log file if it is not empty
   if [[ -s $LOG_NAME ]]; then
-    mkdir -p /home/dnanexus/out/nextflow_log
-    mv "$LOG_NAME" "/home/dnanexus/out/nextflow_log/$LOG_NAME" || true
+    echo "=== Execution complete — upload nextflow log to job output destination ${DX_JOB_OUTDIR%/}"
+    NEXFLOW_LOG_ID=$(dx upload "/home/dnanexus/out/nextflow_log/$LOG_NAME" --path "${DX_JOB_OUTDIR%/}/${LOG_NAME}" --wait --brief --no-progress --parents) &&
+      echo "Upload nextflow log as file: $NEXFLOW_LOG_ID" ||
+      echo "Failed to upload log file of current session $NXF_UUID"
   else
-    echo "No nextflow log file available."
+    echo "=== Execution complete — no nextflow log file available."
   fi
+  rm $LOG_NAME || true
```

Comment:
It probably does not matter as it is uploaded to the platform, but why do we remove the log here? Isn't it redundant?